File: app/src/main/java/com/lee/mvvmproject/MyApplication.java
package com.lee.mvvmproject;

import android.app.Application;

import com.squareup.leakcanary.LeakCanary;

/**
 * Created by Lee.
 * Date : 2019/1/17.
 */
public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        LeakCanary.install(this);
    }
}

File: app/src/main/java/com/lee/mvvmproject/MyApplication.kt
package com.lee.mvvmproject

import android.app.Application
import com.lee.mvvm.utils.AppUtils

import com.squareup.leakcanary.LeakCanary

/**
 * Created by Lee.
 * Date : 2019/1/17.
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        LeakCanary.install(this)
        AppUtils.init(this)
    }
}
